content/guides/output-management.md

```diff
@@ -3,85 +3,262 @@ title: Output Management
 sort: 4
 contributors:
   - skipjack
+  - TheDutchCoder
 ---
 
-Managing webpack's [output](/configuration/output) and including it in your HTML files may not seem tough at first, but once you start [using hashes in filenames](/guides/caching) and outputting [multiple bundles](/guides/code-splitting), things can start to get a bit hairy. However, there's no need to fear as a few plugins exist that will make this process much easier to manage.
+>T This guide extends on code examples found in the [`Asset Management`](/guides/asset-management) guide.
 
-First let's take a look at where you might stand without these plugins:
+So far we've manually included all our assets in our `index.html` file, but as your application grows and once you start [using hashes in filenames](/guides/caching) and outputting [multiple bundles](/guides/code-splitting-libraries), it will be difficult to keep managing your `index.html` file manually. However, there's no need to fear as a few plugins exist that will make this process much easier to manage.
 
-__index.html__
+## Preparation ##
 
-``` html
-<!doctype html>
+First, let's adjust our project a little bit:
+
+__dist/index.html__
+
+``` diff
+  <html>
+    <head>
+-     <title>Asset Management</title>
++     <title>Output Management</title>
++     <script src="./vendor.bundle.js"></script>
+    </head>
+    <body>
+-     <script src="./bundle.js"></script>
++     <script src="./app.bundle.js"></script>
+    </body>
+  </html>
+```
+
+Now adjust the config. We'll be adding a vendor entry point as an example and we'll change the output as well, so that it will dynamically generate bundle names, based on the entry point names:
+
+__webpack.config.js__
+
+``` diff
+  const path = require('path');
+
+  module.exports = {
+    entry: {
+-     index: './src/index.js',
++     app: './src/index.js',
++     vendor: ['lodash']
+    },
+    output: {
+-     filename: 'bundle.js',
++     filename: '[name].bundle.js',
+      path: path.resolve(__dirname, 'dist')
+    }
+  };
+```
+
+Let's run `npm run build` and see what this generates:
+
+``` bash
+Hash: 5f3b0265b87c603b4a0f
+Version: webpack 2.6.1
+Time: 539ms
+           Asset     Size  Chunks                    Chunk Names
+vendor.bundle.js   544 kB       0  [emitted]  [big]  vendor
+   app.bundle.js  2.81 kB       1  [emitted]         app
```

Comment:
That could work, however then the `vendor` bundle wouldn't really be used anywhere. How about we leave `lodash` but don't extract it and instead of `vendor` we add separate entry script that does something else to the page? This way that will be automatically included by the `HTMLWebpackPlugin` (i.e. it should serve as a good way to illustrate the ideas of this article) but doesn't require any extra complexity in terms of plugins or configuration.